pkgs/development/libraries/fplll/default.nix

```
@@ -3,12 +3,12 @@
 }:
 stdenv.mkDerivation rec {
   pname = "fplll";
-  version = "5.2.1";
+  version = "5.3.0";
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = version;
-    sha256 = "015qmrd7nfaysbv1hbwiprz9g6hnww1y1z1xw8f43ysb7k1b5nbg";
+    sha256 = "0wxa4xs7as7y47h7i6prmk5r0srabdvrlkvza3j50pixir5swgvh";
   };
   nativeBuildInputs = [autoconf automake libtool gettext autoreconfHook];
```

the hook will bring in autotool binaries
```suggestion
  nativeBuildInputs = [gettext autoreconfHook];
```